assembly/pom.xml

```diff
@@ -237,51 +238,69 @@
     <profiles>
         <profile>
             <id>cdh5.6.0</id>
-            <properties><extraExclude>org.apache.spark:*</extraExclude></properties>
+            <properties>
+                <excludeSpark>org.apache.spark:*</excludeSpark>
+            </properties>
         </profile>
         <profile>
             <id>cdh5.6.1</id>
-            <properties><extraExclude>org.apache.spark:*</extraExclude></properties>
+            <properties>
+                <excludeSpark>org.apache.spark:*</excludeSpark>
+            </properties>
         </profile>
         <profile>
             <id>cdh5.7.2</id>
-            <properties><extraExclude>org.apache.spark:*</extraExclude></properties>
+            <properties>
+                <excludeSpark>org.apache.spark:*</excludeSpark>
+            </properties>
         </profile>
         <profile>
             <id>cdh5.7.3</id>
-            <properties><extraExclude>org.apache.spark:*</extraExclude></properties>
+            <properties>
+                <excludeSpark>org.apache.spark:*</excludeSpark>
+            </properties>
         </profile>
         <profile>
             <id>cdh5.7.4</id>
-            <properties><extraExclude>org.apache.spark:*</extraExclude></properties>
+            <properties>
+                <excludeSpark>org.apache.spark:*</excludeSpark>
+            </properties>
         </profile>
         <profile>
             <id>cdh5.7.5</id>
-            <properties><extraExclude>org.apache.spark:*</extraExclude></properties>
+            <properties>
+                <excludeSpark>org.apache.spark:*</excludeSpark>
+            </properties>
         </profile>
         <profile>
             <id>cdh5.8.0</id>
-            <properties><extraExclude>org.apache.spark:*</extraExclude></properties>
+            <properties>
+                <excludeSpark>org.apache.spark:*</excludeSpark>
+            </properties>
         </profile>
         <profile>
             <id>cdh5.8.2</id>
-            <properties><extraExclude>org.apache.spark:*</extraExclude></properties>
+            <properties>
+                <excludeSpark>org.apache.spark:*</excludeSpark>
+            </properties>
         </profile>
         <profile>
             <id>cdh5.8.3</id>
-            <properties><extraExclude>org.apache.spark:*</extraExclude></properties>
-        </profile>
-        <profile>
-            <id>cdh5.8.5</id>
```

Comment:
Is 5.8.5 removed by mistake?
I'd rather see this whole part intact. All is needed is <exclude>org.apache.kafka:*</exclude> above.

Reply:
@OlegMazurov must be my mistake. will revert this whole section